pkg/services/datasources/models.go

```diff
@@ -77,6 +79,18 @@ type TeamHTTPHeader struct {
 	Value  string `json:"value"`
 }
 
+// TeamHTTPHeaderValueRegexMatch returns a regex that can be used to check
+// words separated by special characters
+// "namespace!=auth", "env="prod", "env!~dev"
+func (th *TeamHTTPHeader) TeamHTTPHeaderValueRegexMatch() (*regexp.Regexp, error) {
```

Comment:
Is there any reason to have this function as a method on  TeamHTTPHeader? Seems like it might be converted to regular function and moved from models package since it's about data source types.